scripts/compactor_runner.py

```diff
@@ -10,6 +10,10 @@
 /usr/share/corfu/scripts/compactor_runner.py--port <port> --compactorConfig /usr/share/corfu/conf/corfu-compactor-config.yml--freezeCompaction=True
 4. To unfreeze compactor
```

Comment:
It will be helpful to add a comment so people can differentiate between freeze and disabled, and exactly how long the freeze will be.